inspirehep/config.py

```diff
@@ -1496,6 +1504,45 @@ def _(x):
             "default_order": 'desc',
             "order": 2,
         },
+        "core": {
+            "title": 'Core (Desc)',
+            "fields": [
+                {
+                    '_extra_data.relevance_prediction.scores.CORE':
+                        {
+                            "ignore_unmapped": True,
```

Comment:
Deprecated long ago, and probably already removed in ES `2.4`: https://www.elastic.co/guide/en/elasticsearch/reference/1.6/search-request-sort.html#_ignoring_unmapped_fields